README.bg.md

```
@@ -0,0 +1,642 @@
+1. Отидете на [Cerebras](https://www.cerebras.ai/) и натиснете **Get started**. ![step1](docs/images/cerebras/1.png)
+2. Влезте. ![step2](docs/images/cerebras/2.png)
+3. Изберете безплатния план. ![step3](docs/images/cerebras/3.png)
+4. Копирайте ключа в PuriPuly. ![step4](docs/images/cerebras/4.png)
+
+</details>
+
+---
+
+---
+
+### Ако звуковото записване не работи
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)